playbook/app/pb_kits/playbook/pb_label_value/docs/_label_value_props_swift.md

```diff
@@ -0,0 +1,11 @@
+### Props
+| Name | Type | Description | Default | Values |
+| --- | ----------- | --------- | --------- | --------- |
+| **Label** | `String` | Sets the label text |  |  |
+| **Value** | `String` | Sets the value | `nil` |  |
+| **Variant** | `Variant` | Changes the style of the Label Value  | `.default` | `.default` `.details` |
+| **Icon** | `PlaybookGenericIcon` | Adds an icon nexxt to the title | `nil` |  |
+| **Description** | `String` | Sets the description | `nil` |  |
+| **Title** | `String` | Sets the title | `nil` |  |
+| **Date** | `Date` | Adds a date | `nil` |  |
+| **Active** | `Bool` | Changes the style of the Label Value | `false` | `true` `false` |
```

Comment:
PR looks awesome @nickamantia ! Just a question on this file, Since the PR is for the Message kit, wasn't sure if this label_value_props file got included in this PR by mistake? I see you have another open PR for the Label Value kit as well so just making sure!

Reply:
@nidaqg  sorry im just seeing this now! i wish github had better notifications lol. but yes youre correct this file shouldnt be here!!! gonna fix that right now. great catch thank you!!!